src/chara.js

```diff
@@ -394,7 +394,7 @@ var Chara = CreateClass({
     handleSelectEvent: function (key, e) {
         var newState = this.state;
 
-        if (key == "isConsideredInAverage" || key == "EXLBWED") {
+        if (e.target.type === "checkbox") {
```

Comment:
:+1:

`newState[key] = e.target.checked;` also worked

Reply:
Changed in 1594a8f